.hlint.yaml

```diff
@@ -51,3 +51,6 @@
 - ignore:
     name: Avoid restricted qualification
     within: [Oscoin.Protocol.Sync]
+- ignore:
+    name: Avoid restricted qualification
```

Comment:
Can we just get rid of this warning entirely?

Reply:
👍 , I find it not very helpful.